aiplatform/pom.xml

```diff
@@ -68,6 +80,10 @@
       <version>1.4.0</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.google.api.grpc</groupId>
+      <artifactId>grpc-google-cloud-aiplatform-v1</artifactId>
+    </dependency>
```

Comment:
This shouldn't be required. We already have the library in pom.xml

Reply:
I don't see it already in `pom.xml`? This is to bring in the gRPC service definitions, and without the added dependency, the tests do not build.